### PR TITLE
feat(governance): proposal to deploy contract across chain

### PR DESCRIPTION
**Pull request by clemsos**
# Description

This adds a proposal that uses the cross-chain pattern to deploy contract from the DAO to another chain using the Connext bridge 

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Review by julien51 — APPROVED** [review]

File: governance/proposals/014-deploy-up-udt-swap.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+const { ethers } = require('hardhat')
+const { SafeProvider } = require('@safe-global/protocol-kit')
+const { getNetwork } = require('@unlock-protocol/hardhat-helpers')
+const { parseBridgeCall } = require('../helpers/crossChain')
+
+// TODO : change to base
```
Comment (julien51) [inline governance/proposals/014-deploy-up-udt-swap.js:6]: YES!

**Review by julien51 — CHANGES_REQUESTED** [review]
I would be in favor of splitting this in 2 PRs. One for the Connext changes and one for the proposal to deploy the swapping contract so we can point to this more easily!

**Comment by clemsos** [comment]
> I would be in favor of splitting this in 2 PRs. One for the Connext changes and one for the proposal to deploy the swapping contract so we can point to this more easily!

Right. The changes related to bridge have been moved here https://github.com/unlock-protocol/unlock/pull/14204

**Comment by clemsos** [comment]
We didn't need to deploy across chain so closing this

**Pull request closed without merging**